src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -449,7 +449,34 @@ private string UFormatDateString(DateTime dateTime)
                             break;
 
                         case 'V':
-                            sb.Append((dateTime.DayOfYear / 7) + 1);
+                            // .Net Core doesn't implement ISO 8601.
+                            // So we use workaround from https://blogs.msdn.microsoft.com/shawnste/2006/01/24/iso-8601-week-of-year-format-in-microsoft-net/
+                            // with corrections from comments
+
+                            // Culture doesn't matter since we specify start day of week
+                            var calender = CultureInfo.InvariantCulture.Calendar;
+                            var day = calender.GetDayOfWeek(dateTime);
+                            var normalizedDatetime = dateTime;
+
+                            switch (day)
+                            {
+                                case DayOfWeek.Monday:
+                                case DayOfWeek.Tuesday:
+                                case DayOfWeek.Wednesday:
+                                    normalizedDatetime = dateTime.AddDays(3);
+                                    break;
+
+                                case DayOfWeek.Friday:
+                                case DayOfWeek.Saturday:
+                                case DayOfWeek.Sunday:
+                                    normalizedDatetime = dateTime.AddDays(-3);
```

Comment:
In the MSDN blog post, the "cheat" is to add 3 days for Mon-Wed, why are you subtracting 3 days for Fri-Sun?

Reply:
Below in the blog post comments (Keno) contain the fix.
Test values from Wikipedia article pass tests only with the fix.

Reply:
Ok, got it.  Thanks.